### ISSUE TITLE
[Bug] Launch crash with Fabric mod

### ISSUE THREAD

**Issue by TNTxiaoluo** [PojavLauncherTeam/PojavLauncher_iOS#656]
### Describe the bug

When I launch Minecraft fabric, it was crashed
I launched 1.17.1, 1.19, 1.20.1 and 1.20.4 these versions
They were all crashed

### The log file and images/videos

Crash log:https://github.com/PojavLauncherTeam/PojavLauncher_iOS/files/14434794/latestlog.2.txt

### Steps To Reproduce

```markdown
1.Start Pojavlauncher
2.Launch Minecraft Fabric-Loder-0.15.7-1.20.4
3.Crashed
```

### Expected Behavior

I expect you can fix it

### Platform

```markdown
- Device model: iPhone 11 Pro Max
- iOS version: 17.0
- PojavLauncher version: 3.0-development
```

**Comment by ItAnthon** [comment]
you need to add immersive portals 4.2.0

it’s a dependency for another one of your mods. not a bug of ours. it’s user error.

**Comment by ItAnthon** [comment]
[14:42:09] [main/WARN]: Mod resolution failed
[14:42:09] [main/INFO]: Immediate reason: [HARD_DEP_NO_CANDIDATE mini_scaled 4.0.3 {depends immersive_portals @ [>=4.2.0]}, ROOT_FORCELOAD_SINGLE mini_scaled 4.0.3]
[14:42:09] [main/INFO]: Reason: [HARD_DEP mini_scaled 4.0.3 {depends immersive_portals @ [>=4.2.0]}]
[14:42:09] [main/INFO]: Fix: add [add:immersive_portals 4.2.0 ([[4.2.0,âˆž)])], remove [], replace []
[14:42:09] [main/ERROR]: Incompatible mods found!
